api/update-pba/index.ts

```diff
@@ -0,0 +1,33 @@
+import * as express from 'express'
+import { environmentConfig } from '../lib/environment.config'
+import { http } from '../lib/http'
+/**
+ * Handle Update PBA Number
+ *
+ * The response object from PRD is different based on the request object.
+ * In case of just 1 org then we get {org1}
+ *
+ * @param req
+ * @param res
+ * @param next
+ */
+
+async function handleUpdatePBARoute(req: express.Request, res: express.Response, next: express.NextFunction) {
+      try {
+          const {paymentAccounts, orgId} = req.body
+          const updatePbaUrl = `${environmentConfig.services.rdProfessionalApi}/refdata/internal/v1/organisations/${orgId}/pbas`
```

Comment:
Can this be moved to a separate method so that it could be unit tested?
something like getUpdatePbaUrl which takes 1 parameter environmentConfig.services.rdProfessionalApi

Reply:
don't think we do node testing on AO for the time being. once we do the then we will destructure accordingly